blackbox/docs/appendices/release-notes/unreleased.rst

```diff
@@ -42,6 +42,10 @@ Unreleased Changes
 Breaking Changes
 ================
 
+- Arithmetic operations ``*``, ``+`` and ``-`` of types ``integer`` and
```

Comment:
```suggestion
- Changed arithmetic operations ``*``, ``+`` and ``-`` of types ``integer`` and [...]  to throw an exception instead of rolling over from positive to negative or the other way around.
```

Reply:
Thank you, I prefer your wording, it has been changed.